tb_plugin/torch_tb_profiler/profiler/module_op.py

```diff
@@ -0,0 +1,220 @@
+from copy import copy
+
+from .node import ModuleNode, ProfilerStepNode
+from .trace import EventTypes
+
+
+class Module:
```

Comment:
do we need a different data structure for the ops themselves?

Reply:
sorry, I don't quite understand your question.